jboss/container/wildfly/launch/jgroups/test/ha.bats

```diff
@@ -94,7 +94,7 @@ EOF
 @test "Generate JGroups Auth config - missing cluster password" {
   run generate_jgroups_auth_config "" "digest_algo"
   echo "Result: ${output}"
-  [[ "${output}" =~ "No password defined for JGroups cluster." ]]
+  [[ "${output}" =~ .*"No password defined for JGroups cluster.".* ]]
```

Comment:
These are changed manually as well to correct the behavior of =~ operating on a regex. The intent here is that the string contains the message, but does not necessarily match the message completly.